operators/tokenizer/sentencepiece_tokenizer.cc

```diff
@@ -32,35 +33,47 @@ void KernelSentencepieceTokenizer::Compute(const ortc::Tensor<std::string>& inpu
                                            bool add_rev,
                                            ortc::Tensor<int32_t>& output,
                                            ortc::Tensor<int64_t>& output1,
+                                           ortc::Tensor<int32_t>& output2,
                                            std::optional<bool> fairseq) const {
   // Update with the new API
   auto& str_input = input.Data();
   // computation
 
-  std::vector<int64_t> indices;
+  std::vector<int64_t> instance_indices;
```

Comment:
looks great! `instance_indices` should have the offset values as I discussed with Justin. However, we would like to keep `indices` as they were implemented previously as well, in case there are other partner requirements for it - if not, we can deprecate it in the future. could you add back the `indices` output (i.e., we have both `indices` and `instance_indices`)?

i can then approve and we can check this in to unblock shipping for you.

Reply:
Thanks. The old indices are still there just renamed to `instance_indices` to make it clear how they differ from `token_indices`

Reply:
right, sorry missed that. yeah i think the name change should be fine, could you also update it here (and add the output value for `token_indices`): https://github.com/microsoft/onnxruntime-extensions/blob/main/onnxruntime_extensions/_cuops.py#L338

Reply:
Done

Reply:
cheers, thanks! will approve after CI checks pass.